build.d/200-dependencies

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python
+# Copyright 2017 LasLabs Inc.
+
+from os.path import join
+
+from odoobaselib import (
+    addons_config,
+    AUTO_REQUIREMENTS,
+    CUSTOM_DIR,
+    FILE_APT_BUILD,
+    SRC_DIR,
+)
+from odoobaselib.installer import install, INSTALLERS
+
+
+# Build dependencies installed before any others
+install("apt", FILE_APT_BUILD)
+
+for name in INSTALLERS:
+    req_files = []
+    # pip `requirements.txt` files found in repositories
+    if name == "pip" and AUTO_REQUIREMENTS:
+        for repo in addons_config():
```

Comment:
`addons_config` changed its return since #84. I think this should fail as it is.

Anyways, you can be dumber here and just use `req_files += glob.glob(join(SRC_DIR, "*", "requirements.txt"))`